packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -31,13 +31,27 @@ export interface IColor {
   str: string;
 }
 
-export class ColorPicker extends React.Component<IColorPickerProps, any> {
+export class ColorPicker extends React.Component<IColorPickerProps, IColorPickerState> {
   constructor(props: IColorPickerProps) {
     super(props);
 
     this.state = {
       color: getColorFromString(props.color)
-    };
+    } as IColorPickerState;
+  }
+
+  public componentWillReceiveProps(newProps: IColorPickerProps) {
+    const { onColorChanged } = this.props;
+
+    if (newProps.color !== undefined && newProps.color !== this.state.color.str) {
```

Comment:
I think it might be better to have this be if (newProps.color && newProps.color !== this.state.color.str)